graphql2/graphqlapp/contactmethod.go

```diff
@@ -71,6 +71,10 @@ func (m *Mutation) CreateUserContactMethod(ctx context.Context, input graphql2.C
 }
 
 func (m *Mutation) UpdateUserContactMethod(ctx context.Context, input graphql2.UpdateUserContactMethodInput) (bool, error) {
+	if input.Value != nil {
+		return false, validation.NewFieldError("Value", "changing contact method value is not allowed")
```

Comment:
Should we also be doing this for Type then — since this is disabled on the UI? From what it looks like here, they should behave the same both here and on the UI, since they would both be disabled fields, and both be un-editable when conversing with GraphQL

Also, is this validation needed since we are covering the field errors within the store below? Only one would ever return

Reply:
Type isn't in the schema so it isn't needed here. But good catch on the value.

The old UI is going to send the `value` field always, so we should delegate to the store and only reject it if it changes.